apps/condo/domains/organization/integrations/sbbol/SbbolSecretStorage.js

```diff
@@ -38,48 +38,48 @@ class SbbolSecretStorage {
         await this.#setValue('clientSecret:updatedAt', dayjs().toISOString())
     }
 
-    async getAccessToken () {
-        return this.#getValue('accessToken')
+    async getAccessToken (userId) {
+        return this.#getValue(`${userId}:accessToken`)
     }
 
-    async setAccessToken (value, options) {
-        await this.#setValue('accessToken', value, options)
-        await this.#setValue('accessToken:updatedAt', dayjs().toISOString())
+    async setAccessToken (value, userId, options) {
+        await this.#setValue(`${userId}:accessToken`, value, options)
+        await this.#setValue(`${userId}:accessToken:updatedAt`, dayjs().toISOString())
     }
 
-    async isAccessTokenExpired () {
-        return await this.#isExpired('accessToken')
+    async isAccessTokenExpired (userId) {
+        return await this.#isExpired(`${userId}:accessToken`)
     }
 
-    async getRefreshToken () {
-        return this.#getValue('refreshToken')
+    async getRefreshToken (userId) {
+        return this.#getValue(`${userId}:refreshToken`)
     }
 
-    async setRefreshToken (value) {
+    async setRefreshToken (value, userId) {
         const options = { expiresAt: dayjs().add(REFRESH_TOKEN_TTL_DAYS, 'days').unix() }
-        await this.#setValue('refreshToken', value, options)
-        await this.#setValue('refreshToken:updatedAt', dayjs().toISOString())
+        await this.#setValue(`${userId}:refreshToken`, value, options)
+        await this.#setValue(`${userId}:refreshToken:updatedAt`, dayjs().toISOString())
     }
 
-    async isRefreshTokenExpired () {
-        return await this.#isExpired('refreshToken')
+    async isRefreshTokenExpired (userId) {
+        return await this.#isExpired(`${userId}:refreshToken`)
     }
 
-    async setOrganization (id) {
-        await this.#setValue('organization', id)
+    async setOrganization (id, userId) {
+        await this.#setValue(`${userId}:organization`, id)
```

Comment:
This is organization that corresponds to our organization as a partner of SBBOL.
It is not in scope of user.
This way it's easier to figure out to what Organization `clientSecret` and `clientId` belongs.
```suggestion
    async setOrganization (id) {
        await this.#setValue(`organization`, id)
```